pkg/kp/service.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/square/p2/pkg/pods"
 )
 
+var (
+	// prints the HTTP response on stderr, while exiting 0 if the status code is 200
+	httpStatusCheck  = `if [[ $(curl http://$(hostname):%v/_status -s -o /dev/stderr -w "%%{http_code}") == "200" ]] ; then exit 0 ; else exit 2; fi`
+	httpsStatusCheck = `if [[ $(curl https://$(hostname):%v/_status -s -o /dev/stderr -w "%%{http_code}" --cacert ${SECRETS_PATH}/service2service.ca.pem) == "200" ]] ; then exit 0 ; else exit 2; fi`
```

Comment:
This is pretty nasty as is though. Right now, p2 users will run p2-bootstrap believing that they have set up a functioning cluster but will immediately crash on finding that they 1) haven't exported a SECRETS_PATH and 2) don't have any need for a custom certificate authority.

Ideally Consul would offer an option to provide a CA path when doing http checks. Looking at the Consul source this seems relatively easy to do. 

In the meantime I'd prefer to make this path configurable in the preparer instead.